test/multi_tenancy.sh

```diff
@@ -126,12 +129,14 @@ function test_user_has_proper_access() {
     local xfuser='-H x-forwarded-user:'"$user"
     local xocpns='-H x-ocp-namespace:'"$user_project_list"
     local xfroles='-H x-forwarded-roles:project_user'
+    local xff='-H x-forwarded-for:localhost'
```

Comment:
does this get expanded automatically? or will the config reject it since it expects `::1|127.0.0.1`?

Reply:
the header gets compared against the regex `.*` , and is used in OD to resolve the address by creating instance of `InetSocketAddress`. So the value passed here must be usable as hostname parameter of `InetSocketAddress`.